foundation-extensions/profile/README.md

```diff
@@ -14,7 +14,19 @@ To get started with the Profile extension:
    * Update user attributes.
    * Remove user attributes.
 
-## Add Profile to your App
+## Install the Adobe Experience Platform Profile extension in Experience Platform Launch
+
+1. In Experience Platform Launch, in your mobile property, click the **Extensions** tab.
```

Comment:
"In the Data Collection UI, in your mobile property, select the **Extensions** tab."

(Using click goes against usability guidelines - https://experienceleague.adobe.com/docs/authoring-guide-exl/using/authoring/style-guide/authoring-style-guide.html - I know this is for ExL, but the same guidelines apply for all the documentation.)